### PR TITLE
Modulesync 5.1.0

### PR DESCRIPTION
**Pull request by smortex**

### CODE REVIEW

File: .fixtures.yml [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
-    apt:
-      repo: puppetlabs/apt
-      ref: 2.0.0
-    elastic_stack:
-      repo: elastic/elastic_stack
-      ref: 6.1.0
-    stdlib:
-      repo: puppetlabs/stdlib
-      ref: 4.21.0
+    apt: puppetlabs/apt
```
Comment (bastelfreak) [inline .fixtures.yml:3]: can you please update this to git urls? we always test against master for unit tests and latest forge release in acceptance tests.

File: ; [hunk 2]
```diff
@@ -0,0 +1,37 @@
+Modulesync 5.1.0
```
Comment (bastelfreak) [inline ;:1]: @smortex what's that?

File: .puppet-lint.rc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+--fail-on-warnings
+--no-parameter_documentation-check
```
Comment (bastelfreak) [inline .puppet-lint.rc:2]: it looks like we can opt-in to those two? In that case add the following to the .sync.yml
```
---
.puppet-lint.rc:
  enabled_lint_checks:
    - parameter_documentation
    - parameter_types
```

File: ; (hunk 2 above)
Comment (smortex) [reply ;:1]: :facepalm:  Saving a commit message using `:wq` while learning to use a dvorak layout became `:w;` :unamused:

File: .fixtures.yml (hunk 1 above)
Comment (smortex) [reply .fixtures.yml]: Done! Will do the same for the other ES modules.

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**